### PR TITLE
store limit v2: add more experimental description

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: configure-store-limit.md [hunk 1]
```diff
@@ -72,4 +72,6 @@ store limit 1 5 remove-peer         // 设置 store 1 删除 peer 的速度上
 
 当 [`store-limit-version`](/pd-configuration-file.md#store-limit-version-从-v710-版本开始引入) 设置为 `v2` 时，Store Limit v2 生效。在此模式下，Operator 调度限制将根据 TiKV Snapshot 执行情况进行动态调整。当 TiKV 积压的任务较少时，PD 会增加其调度任务。相反，PD 会减少对该节点的调度任务。此时，你无需关注如何设置 `store limit` 以加快调度进度。
 
-在该模式下，TiKV 执行速度成为迁移进度的主要瓶颈。你可以通过 **TiKV Details** > **Snapshot** > **Snapshot Speed** 面板判断当前调度速度是否达到 TiKV 限流设置。通过调整 TiKV Snapshot Limit ([`snap-io-max-bytes-per-sec`](/tikv-configuration-file.md#snap-io-max-bytes-per-sec)) 来增加或减少该节点的调度速度。
\ No newline at end of file
+在该模式下，TiKV 执行速度成为迁移进度的主要瓶颈。你可以通过 **TiKV Details** > **Snapshot** > **Snapshot Speed** 面板判断当前调度速度是否达到 TiKV 限流设置。通过调整 TiKV Snapshot Limit ([`snap-io-max-bytes-per-sec`](/tikv-configuration-file.md#snap-io-max-bytes-per-sec)) 来增加或减少该节点的调度速度。
+
+在当前版本中，将该配置项设置为 `"v2"` 为实验特性，不建议在生产环境中使用。
```
Comment (lhy1024) [inline configure-store-limit.md:77]: from https://github.com/pingcap/docs-cn/blob/148b399f97d5a9abec18733b0303b510297553c2/pd-configuration-file.md?plain=1#L430

File: configure-store-limit.md [hunk 2]
```diff
@@ -72,4 +72,6 @@ store limit 1 5 remove-peer         // 设置 store 1 删除 peer 的速度上
```
Comment (Oreoxmt) [inline configure-store-limit.md:72]:
```suggestion

> **警告：**
>
> Store Limit v2 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: configure-store-limit.md (hunk 1 above)
Comment (Oreoxmt) [inline configure-store-limit.md:77]:
```suggestion
```

File: pd-control.md [hunk 3]
```diff
@@ -454,7 +454,7 @@ config show cluster-version
 
 - `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（弃用）。
 
-- `store-limit-version` 用于设置 `store limit` 限制模式，目前提供两种方式：`v1` 和 `v2`。默认值为 `v1`。在 `v1` 模式下，你可以手动修改 `store limit` 以限制单个 TiKV 调度速度。`v2` 模式为实验特性，在 `v2` 模式下，你无需关注 `store limit` 值，PD 将根据 TiKV Snapshot 执行情况动态调整 TiKV 调度速度。详情请参考 [Store Limit v2 原理](/configure-store-limit.md#store-limit-v2-原理)。
+- `store-limit-version` 用于设置 `store limit` 限制模式，目前提供两种方式：`v1` 和 `v2`。默认值为 `v1`。在 `v1` 模式下，你可以手动修改 `store limit` 以限制单个 TiKV 调度速度。`v2` 模式为实验特性，在 `v2` 模式下，你无需关注 `store limit` 值，PD 将根据 TiKV Snapshot 执行情况动态调整 TiKV 调度速度。详情请参考 [Store Limit v2 原理](/configure-store-limit.md#store-limit-v2-原理)。在当前版本中，将该配置项设置为 `"v2"` 为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline pd-control.md:457]:
```suggestion
- `store-limit-version` 用于设置 `store limit` 限制模式，目前提供两种方式：`v1` 和 `v2`。默认值为 `v1`。在 `v1` 模式下，你可以手动修改 `store limit` 以限制单个 TiKV 调度速度。`v2` 模式为实验特性，不建议在生产环境中使用。在 `v2` 模式下，你无需关注 `store limit` 值，PD 将根据 TiKV Snapshot 执行情况动态调整 TiKV 调度速度。详情请参考 [Store Limit v2 原理](/configure-store-limit.md#store-limit-v2-原理)。
```

File: configure-store-limit.md [hunk 4]
```diff
@@ -70,6 +70,9 @@ store limit 1 5 remove-peer         // 设置 store 1 删除 peer 的速度上
 
 ## Store Limit v2 原理
 
+> **警告：**
+>
+> Store Limit v2 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
```
Comment (Oreoxmt) [inline configure-store-limit.md:75]:
```suggestion
> Store Limit v2 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: releases/release-7.1.0.md [hunk 5]
```diff
@@ -386,7 +386,7 @@ TiDB 7.1.0 为长期支持版本 (Long-Term Support Release, LTS)。
 
 + PD
 
-    - 新增基于 snapshot 执行细节来自动调整 store limit 大小的控制器。将 `store-limit-version` 设置为 `v2` 即可开启该控制器，开启后，用户无需手动调整 `store limit` 配置来控制扩缩容的速度 [#6147](https://github.com/tikv/pd/issues/6147) @[bufferflies](https://github.com/bufferflies)
+    - 新增基于 snapshot 执行细节来自动调整 store limit 大小的控制器（实验特性）。将 `store-limit-version` 设置为 `v2` 即可开启该控制器，开启后，用户无需手动调整 `store limit` 配置来控制扩缩容的速度 [#6147](https://github.com/tikv/pd/issues/6147) @[bufferflies](https://github.com/bufferflies)
```
Comment (lhy1024) [inline releases/release-7.1.0.md:389]: @Oreoxmt PTAL

File: configure-store-limit.md (hunk 1 above)
Comment (Oreoxmt) [reply configure-store-limit.md]: > This feature is experimental between 7.1 and 8.5, and will GA in 8.5

@lhy1024 I recommend:

1. Changing the base of this PR to `release-8.4`. I will cherry-pick this PR to v7.1 and later versions.
2. Creating a new PR to upadte the `master` branch for v8.5 changes: removing the experimental note in https://github.com/pingcap/docs-cn/blob/3c0975ec627e0328664da94daf5efda37d6fb562/pd-control.md?plain=1#L457

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by niubell — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**